pysparkling/sql/expressions/mappers.py

```diff
@@ -4,13 +4,11 @@
 import re
 import string
 
-from pysparkling.sql.expressions.expressions import Expression, NullSafeColumnOperation, \
-    UnaryExpression
+from pysparkling.sql.expressions.expressions import Expression, NullSafeColumnOperation, UnaryExpression
 from pysparkling.sql.internal_utils.column import resolve_column
-from pysparkling.sql.types import create_row, StringType
```

Comment:
Grr. Ok, I thought the `case_sensitive` flag would fix it, but it seems this is also due to the `order_by_type` flag. Can you set that to false so that this line is ordered alphabetical irrespective of capitalization/type. 

Everything else is checked off on my side. Thanks!

Reply:
Ok, done.